Update IdP docs (#98)
netbirdio/docs: src/pages/selfhosted/identity-providers.mdx

    <img src="/docs-static/img/integrations/identity-providers/self-hosted/keycloak-add-role.png" alt="high-level-dia" className="imagewrapper"/>
</p>

<Note>
Optional

> Netbird offers the ability to automatically delete a user from the Keycloak side when the user is deleted from the associated account.

Reviewer: ```suggestion
NetBird offers the ability to automatically delete a user from the Keycloak side when the user is deleted from the associated account.
```

This guide is a part of the [NetBird Self-hosting Guide](/selfhosted/selfhosted-guide) and explains how to integrate 
**self-hosted** NetBird with [Google Workspace](https://workspace.google.com/).

<Note>

> Beginning with NetBird version v0.23.6 and onwards, the Google Workspace IdP manager no longer necessitates the creation of a custom admin role called `User and Schema Management`.
> Instead, we are transitioning towards a more tailored role designed specifically for managing read-only user information.
> Consequently, you have the option to remove the previously established custom admin role and refer to the documentation to correctly configure the admin role scope for read-only access.

Reviewer: ```suggestion
Beginning with NetBird version v0.23.6 and onwards, the Google Workspace IdP manager no longer requires the creation of a custom admin role called `User and Schema Management`. 
Instead, we are transitioning towards a more tailored role explicitly designed for managing read-only user information. 
Consequently, you have the option to remove the previously established custom admin role and refer to the documentation to configure the admin role scope for read-only access correctly.
```

    <img src="/docs-static/img/integrations/identity-providers/self-hosted/auth0-machine-authorization.png" alt="high-level-dia" className="imagewrapper"/>
</p>

<Note>
Optional

Netbird offers the ability to automatically delete a user from the Auth0 side when the user is deleted from the associated account.

>  To enable this functionality, simply include the `--user-delete-from-idp` flag in the management startup command within your Docker Compose configuration. If you choose to enable this feature,
>  please ensure that you assign the `delete:users` permission following the steps outlined above.

Reviewer: ```suggestion
To enable this functionality, include the `--user-delete-from-idp` flag in the management startup command within your Docker Compose configuration. If you choose to enable this feature, please ensure that you assign the `delete:users` permission following the steps outlined above.
```
